frmts/georaster/georaster_dataset.cpp

```diff
@@ -1306,13 +1301,7 @@ GDALDataset *GeoRasterDataset::Create(const char *pszFilename, int nXSize,
         poGRD->poGeoRaster->bGenStatsUseSamplingWindow = true;
     }
 
-    pszFetched = CSLFetchNameValue(papszOptions, "GENSTATS_HISTOGRAM");
-
-    if (pszFetched != nullptr)
-    {
-        bGenStatsOptionsUsed = true;
-        poGRD->poGeoRaster->bGenStatsHistogram = EQUAL(pszFetched, "TRUE");
-    }
+    poGRD->poGeoRaster->bGenStatsHistogram = CPLFetchBool(papszOptions, "GENSTATS_HISTOGRAM", FALSE);
```

Comment:
hum I realize my suggestion to use CPLFetchBool might cause issues here as bGenStatsOptionsUsed = true will no longer be set.

But I also realize that autotest/gdrivers/georaster.py hasn't been updated. It could make sense for long-term maintenance to test that new functionality